lib/rules/use-brace-expansion.js

```diff
@@ -28,15 +28,18 @@ module.exports = {
       CallExpression(node) {
         if (!ember.isComputedProp(node)) return;
 
+        const hasBraces = arr => !!arr.join('.').match(/\{.*\}/g);
```

Comment:
After the refactor I did to fix the failing tests that I added, having `hasBraces` accept the array is definitely more convenience now, since I can pass it directly to `filter`.